src/pages/about-us.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import Layout from '@components/Layout';
+import AboutUsCover from '@sections/AboutUsCover';
+import AboutUsContent from '@sections/AboutUsContent';
+import { graphql } from 'gatsby';
+
+export interface ISanityAboutUsItem {
```

Comment:
What is the `I` preffix? "Interface"? Why not only `SanityAboutUsItem` ?

Reply:
No special reason, I'll remove the `I`